test/unittests/test_recordstore.py

```diff
@@ -457,6 +461,9 @@ def test_create_project(self):
     def test_project_info(self):
         self.assertEqual(self.store.project_info("TestProject")["name"], "TestProject")
 
+    def test_clear(self):
+        pass  # raises UserWarning
```

Comment:
Have you pulled/fetched my fork (apdavison/sumatra)? If not, I'll just amend the previous commit and force-push, since it's such a tiny change.